GPUVerifyLib/GPUVerifyErrorReporter.cs

```diff
@@ -527,12 +527,12 @@ private static string GetStateName(AssertCounterexample assertCex)
             return GetStateName(assertCex.FailingAssert.Attributes, assertCex);
         }
 
-        private static string GetSourceFileName()
+        protected static string GetSourceFileName()
```

Comment:
Exposing this seems a bit dubious. It assumes that the way we handle the command line is identical to the way you handle it. It might be better to define a `GetSourceFileName` method on yourself and that you are in complete control over.